### ISSUE TITLE
Modula lexer: multiline comment sometimes failed to correctly highlight

### ISSUE THREAD

**Issue by 1997tn** [stefankueng/BowPad#310]
On Modula-2 and Modula-3, they only have `(*` and `*)` for comment (both single line or multiline) unlike Pascal also has `{ }` and `//`. Like the title, sometimes it failed to correctly highlight multiline comment, for example, you typed `(*` and expect the lines following it to be highlighted as commented but they are not.

**Comment by stefankueng** [comment]
created an issue in the right place: https://github.com/ScintillaOrg/lexilla/issues/128

and please, next time just attach the file in question here. I don't like to download huge zips and have to extract and then search for a single small file.

**Comment by stefankueng** [comment]
example please

**Comment by 1997tn** [comment]
> example please

https://github.com/modula3/cm3/tree/master/examples

Open a random source file, try to put (* arbitrarily, sometimes you will see only the current line got highlighted as comment, the expected behavior is all lines following it (until a *) is found).

**Comment by stefankueng** [comment]
so you expect me to go through hundreds of files, find one that's actually a modula file, then try around until I get the problem.

Why can't you just attach a file with your comment starts added here that shows the problem?

**Comment by 1997tn** [comment]
> so you expect me to go through hundreds of files, find one that's actually a modula file, then try around until I get the problem.
> 
> Why can't you just attach a file with your comment starts added here that shows the problem?

No, I don't expect you to do that. I tell you to pick _only_ a random source file (.i3 or .m3 extension). I discovered this bug on the process of coding, I have no particular source file for you to reproduce it. What I described is a dynamic process (trying to put (* without *) to find out if it correctly mark the following lines as comment or not), you expect me to give you a static source file to reproduce it. I can't.
